README.md

```diff
@@ -87,9 +98,7 @@ from root directory via `make tests` command, otherwise use standard PyTest synt
 https://universum.readthedocs.io/en/latest/configuring.html). Usage of such files
 is illustrated in `run_basic_example.sh` script.
 
-`analyzers` directory is not a part of Universum itself. It contains [example external scripts](
-https://universum.readthedocs.io/en/latest/code_report.html) compatible with Universum
-for static (and other types of) analysis.
+`universum_log_collapser` is a directory with Java sources of Jenkins plugin for Universum pretty display.
```

Comment:
I think it would look better with reverse order of words, like this:
`universum_log_collapser` contains Universum Jenkins plugin sources. The plugin implements coloring of failed steps and provides collapsing/expansion of log sections.